src/main/java/com/uber/cadence/workflow/WorkflowInterceptor.java

```diff
@@ -51,6 +51,10 @@ public Promise<WorkflowExecution> getWorkflowExecution() {
     }
   }
 
+  //to match behavior in go client: interceptor executeWorkflow method
+  byte[] executeWorkflow(SyncWorkflowDefinition workflowDefinition, byte[] input)
```

Comment:
instead of raw input, we might want to create a separate entity to include WorkflowStartEventAttributes